From 4fb027f4664b0571b41bd22a8921940b4b5bfb4b Mon Sep 17 00:00:00 2001
From: Ben Hutchings <ben@decadent.org.uk>
Date: Sat, 7 May 2011 04:26:50 +0100
Subject: [PATCH] slub: Reenable for m68k

Following commit 4aac0b4815ba592052758f4b468f253d383dc9d6
'm68k/mm: Set all online nodes in N_NORMAL_MEMORY', SLUB should
actually work.
---
 init/Kconfig |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/init/Kconfig b/init/Kconfig
index 47dd02f..f108e91 100644
--- a/init/Kconfig
+++ b/init/Kconfig
@@ -1209,7 +1209,7 @@ config SLAB
 	  per cpu and per node queues.
 
 config SLUB
-	depends on BROKEN || NUMA || !DISCONTIGMEM
+	depends on BROKEN || NUMA || !DISCONTIGMEM || M68K
 	bool "SLUB (Unqueued Allocator)"
 	help
 	   SLUB is a slab allocator that minimizes cache line usage
-- 
1.7.4.4

